tumblr-photo-video-ripper.py

```diff
@@ -61,15 +61,15 @@ def __init__(self, queue, proxies=None):
 
     def run(self):
         while True:
-            medium_type, post, target_folder = self.queue.get()
-            self.download(medium_type, post, target_folder)
+            site, medium_type, post, target_folder = self.queue.get()
```

Comment:
I don't think we need to add `site` here. Actually `post` has already contains all the needed data, including links and etc.

Reply:
It's a lot easier to follow by just threading through the site info into the queue entry then untuple it here to access it. Otherwise its not clear where the site info comes from due to the lack of explicit types on python.

Reply:
I'm okay with this.